File: apps/web/public/bridging/Day3/C/mini_project.c
/*
 * Day 3 – Mini project: simple menu program (C)
 * Menu: add, subtract, or exit.
 * Compile: gcc -o mini_project mini_project.c
 * Run:     ./mini_project
 */
#include <stdio.h>

int main(void) {
    int choice;
    int running = 1;

    while (running) {
        printf("\n--- Simple Calculator ---\n");
        printf("1. Add two numbers\n");
        printf("2. Subtract two numbers\n");
        printf("3. Exit\n");
        printf("Choice (1-3): ");
        scanf("%d", &choice);

        if (choice == 1) {
            int a, b;
            printf("First number: ");
            scanf("%d", &a);
            printf("Second number: ");
            scanf("%d", &b);
            printf("Result: %d\n", a + b);
        } else if (choice == 2) {
            int a, b;
            printf("First number: ");
            scanf("%d", &a);
            printf("Second number: ");
            scanf("%d", &b);
            printf("Result: %d\n", a - b);
        } else if (choice == 3) {
            printf("Bye!\n");
            running = 0;
        } else {
            printf("Invalid choice.\n");
        }
    }

    return 0;
}

File: apps/web/public/bridging/Day3/Java/MiniProject.java
/**
 * Day 3 – Mini project: simple menu calculator (Java)
 * A small menu-driven program: add, subtract, or exit.
 * Compile: javac MiniProject.java
 * Run:     java MiniProject
 */
import java.util.Scanner;

public class MiniProject {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        boolean running = true;

        while (running) {
            System.out.println("\n--- Simple Calculator ---");
            System.out.println("1. Add two numbers");
            System.out.println("2. Subtract two numbers");
            System.out.println("3. Exit");
            System.out.print("Choice (1-3): ");
            int choice = sc.nextInt();

            if (choice == 1) {
                System.out.print("First number: ");
                int a = sc.nextInt();
                System.out.print("Second number: ");
                int b = sc.nextInt();
                System.out.println("Result: " + (a + b));
            } else if (choice == 2) {
                System.out.print("First number: ");
                int a = sc.nextInt();
                System.out.print("Second number: ");
                int b = sc.nextInt();
                System.out.println("Result: " + (a - b));
            } else if (choice == 3) {
                System.out.println("Bye!");
                running = false;
            } else {
                System.out.println("Invalid choice.");
            }
        }
        sc.close();
    }
}
